src/vstest.console/Program.cs

```diff
@@ -36,7 +37,26 @@ public static int Main(string[] args)
                 System.Diagnostics.Debugger.Break();
             }
 
+            SetCultureSpecifiedByUser();
+
             return new Executor(ConsoleOutput.Instance).Execute(args);
         }
+
+        private static void SetCultureSpecifiedByUser()
```

Comment:
DRY.

Reply:
Specify where it should be common for all these 3 different application